### ISSUE TITLE
[SUGGESTION] An option to disable ship rename and 30 minute reminder

### ISSUE THREAD

**Issue by Pawnp** [tsunamayo/Starship-EVO#5197]
What it says exactly on the tin.
I'm sure many would appreciate these reminders and warnings, but a way to disable them would be nice for the folks who would rather not see them.

**Comment by tsunamayo** [comment]
Hi, I can flag them beginner so they are not shown if you toggle off beginner help message in F1 menu. Waiting to see what other people think of that.

**Comment by TIKIRobo** [comment]
I wouldn't mind if i could turn them off individually, I really like the 30 minute reminder as it keeps me on track but ive played the game enough that I dont need the ship rename warning anymore

**Comment by Britanniczilla** [comment]
Yeah personally I'd also like either an option to reduce the opacity of the notification/notifications in general, move it to either the top left or right corner of the screen, or just have it entirely toggleable.

**Comment by asanagisae** [comment]
The "30 minute export ship" reminder regularly gets in the way of building. For some reason, the warning pops up rather often when I'm trying to change the blocks in my toolbars, and it completely obscures rows 1 and 2.

The changes I make to my builds during each play session are often quite significant and involved, regularly taking 1-3 hours to complete. These are the kinds of changes I _don't want to save_ until I'm absolutely 100% sure they're working, since fixing them later when they're baked into the blueprint will be a much bigger headache.

I turned on the setting for "Skip newcomer messages" and I think this should qualify. Perhaps a different solution would be to have a configuration section for all game-related messages, so you can turn them on/off individually.

![20240207143319_1](https://github.com/tsunamayo/Starship-EVO/assets/10069469/591df29f-0d0e-47c3-94e2-b03e42e0a87b)
